follow-ups/arrays-objects/css/custom.css

```diff
@@ -0,0 +1,346 @@
+@font-face {
+  font-family: 'FontAwesome';
+  src: url('../lib/font/font-awesome/fontawesome-webfont.eot?v=4.5.0');
+  src: url('../lib/font/font-awesome/fontawesome-webfont.eot?#iefix&v=4.5.0') format('embedded-opentype'), url('../lib/font/font-awesome/fontawesome-webfont.woff2?v=4.5.0') format('woff2'), url('../lib/font/font-awesome/fontawesome-webfont.woff?v=4.5.0') format('woff'), url('../lib/font/font-awesome/fontawesome-webfont.ttf?v=4.5.0') format('truetype'), url('../lib/font/font-awesome/fontawesome-webfont.svg?v=4.5.0#fontawesomeregular') format('svg');
```

Comment:
Ta linia nie musi być taka długa ;)
Można to podzielić np. po przecinkach i łatwiej się będzie robić review... bez przewijania w poziomie
nie mówiąc już o pracy w terminalu